specs/protocol/jovian/derivation.md

```diff
@@ -10,17 +10,33 @@
 
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 
-The Jovian network upgrade timestamp defines the timestamp at which all functionality in this document is considered the
-consensus rules for an OP Stack based network.
+## Activation Block Rules
 
-The upgrade transaction details below are based on a branch at commit hash 7daaabbf2c2cce892aa171bc7e1331ad31bcc8ca, and
-will be updated once a contracts release is made.
+The first block with a timestamp at or after the Jovian activation time is considered the _Jovian activation block_.
 
-On the activation block, a set of deposit transaction based upgrade transactions are deterministically generated by the
-derivation pipeline in the following order:
+To not modify or interrupt the system behavior regarding gas computations, the activation block must not include any
+non-deposit transactions. Sequencer must enforce this by setting `noTxPool` to `true` in the payload attributes. This
+rule must be checked during derivation at the batch verification stage, and if the batch for the activation block
+contains any transactions, it must be `DROP`ped following the batch-dropping rules introduced in the
+[Holocene upgrade span-batch rules](../holocene/derivation.md#span-batches).
```

Comment:
Did you link to the right section here? Span batches are optional, what would the rule be for singular batches?

Reply:
Decided to just remove this reference. It's implicitly clear that Holocene rules just apply post-Holocene, no need to mention it again.